lib/scrabble.rb

```diff
@@ -15,4 +14,54 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
+    score = 0
+    if word != nil
+      word_array = word.chars
+      word_array.each do | letter |
+        score += point_values[letter.upcase]
+      end
+    end
+    score
+  end
+
+  def  score_with_multipliers(word, multipliers, total_multiplier = 1)
+    score = 0
+    if word != nil
```

Comment:
Would probably have reversed the conditional here, thinking in negatives is suboptimal